xamarin/realm-tutorial-dotnet/LoginPage.xaml.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using AsyncTask = System.Threading.Tasks.Task;
```

Comment:
I agree. Since all the other apps are using "Task" as the Realm model, and we want consistency, the other option I think I have is to use System.Threading.Tasks.Tasks in each method signature. You think that's a better approach?

Reply:
I also think renaming your model Task to TaskItem or similar would make more sense :)

Reply:
I think that would be more confusing for the inexperienced Realm dev -- "why do we call it a TaskItem here but store it in Realm as a Task?"

Reply:
Actually, this is a moot point. I only use the AsyncTask remapping in this class, and we don't reference the Realm Task in here, so there's no class conflict.